src/main/java/com/test/KafkaTestApp/client/ClientController.java

```diff
@@ -0,0 +1,25 @@
+package com.test.KafkaTestApp.client;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("clients")
+public class ClientController {
+
+  private final ClientManager clientManager;
+
+  public ClientController(ClientManager clientManager) {
+    this.clientManager = clientManager;
+  }
+
+  @PostMapping
+  @ResponseStatus(value = HttpStatus.CREATED)
+  public void create(@RequestBody Client client) {
+    clientManager.create(client);
```

Comment:
would ask you to add a test that will include checking, that the msgs was produced into kafka.